.github/workflows/ci.yml

```diff
@@ -4,6 +4,9 @@ env:
   # come automatically. If the version specified here is no longer the latest stable version,
   # then please feel free to submit a PR that adjusts it along with the potential clippy fixes.
   RUST_STABLE_VER: "1.76" # In quotes because otherwise (e.g.) 1.70 would be interpreted as 1.7
+  # If updating, ensure that it is also updated in Cargo.toml
+  # Explanation of the version choice
+  MINIMUM_SUPPORTED_RUST_VERSION: "1.70"
```

Comment:
I wouldn't expect this to do anything without the MSRV jobs being added

Reply:
I think that's fair, but it's going to be there afterward anyhow so. :shrug:

Reply:
Adding the jobs should probably be its own PR, I think it's ok to add this here though (that we at least know what the current MSRV would be), but well yeah it doesn't do anything.

Reply:
Removed for now, since controversial.